README.md

```diff
@@ -57,9 +58,10 @@ Plugin    |Backend
 `kaniko`  |[kaniko](https://github.com/GoogleCloudPlatform/kaniko)                        |Yes ✅    |                 |             |
 `img`     |[img](https://github.com/genuinetools/img)                                     |Yes ✅    |                 |             |
 `gcb`     |[Google Cloud Container Builder](https://cloud.google.com/container-builder/)  |Yes ✅    |Yes ✅           |             |
+`acb`     |[Azure Container Registry Build](https://azure.microsoft.com/services/container-registry/)|Yes ✅    |                 |             |
```

Comment:
BuildKit LLB is planned here, and we have our own form of `cloudbuild.yaml` planned as well

Reply:
Thanks :+1:

Is it planned to be `cloudbuild.yaml`-compatible, or something similar but different?

Reply:
Not compatible atm, so it'd be more like `acb.yml`. Secrets and such will be Azure focused, cloudbuild is GCP focused. Overall it should be a super set of functionality WRT the step processing.

An example - currently it's in toml format but will probably be switched:
https://raw.githubusercontent.com/Azure/acr-builder/master/templating/testdata/helloworld/git-build.toml